common/src/main/java/com/sequenceiq/cloudbreak/cloud/storage/LocationHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.cloudbreak.cloud.storage;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class LocationHelper {
+
+    private static final Pattern EXTRACT_BUCKET_PATTERN = Pattern.compile("^s3.?://([^/]*)");
```

Comment:
Not a big deal but I suggest replacing `*` with `+`.

Reply:
Done.